ibm_catalog.json

```
@@ -251,6 +251,11 @@
                       },
                       {
                         "key":"instance_cbr_rules"
+                      },
+                      {
+                        "key":"prefix",
+                        "required":true,
+                        "description": "Prefix to append to all resources created by this solution. Allowed values are either a string, an empty string (\"\") or __NULL__."
```

You can't enter an empty string in projects UI when `"required":true` is set, so leave that out of the catalog description. Also "append" is the wrong word here - that means add to the end of. Maybe update to this:

```suggestion
                        "description": "Prefix to add to all resources created by this solution. To not use any prefix value, you can enter the string `__NULL__`."
```

---

Done.